src/main/java/org/apache/maven/shared/verifier/Verifier.java

```diff
@@ -1237,20 +1237,9 @@ public void executeGoals( List<String> goals, Map<String, String> envVars )
 
         File logFile = new File( getBasedir(), getLogFileName() );
 
-        for ( Object cliOption : cliOptions )
+        for ( String cliOption : cliOptions )
         {
-            String key = String.valueOf( cliOption );
-
-            String resolvedArg = resolveCommandLineArg( key );
-
-            try
-            {
-                args.addAll( Arrays.asList( CommandLineUtils.translateCommandline( resolvedArg ) ) );
-            }
-            catch ( Exception e )
-            {
-                e.printStackTrace();
-            }
+            args.add( resolveCommandLineArg( cliOption ) );
```

Comment:
I believe that most quoting here is not necessary anymore: `resolveCommandLineArg()`

Reply:
`resolveCommandLineArg()` is responsible for replace `${basedir}` placeholder in arguments

Reply:
I know, but why is there so much fiddling afterwards. I if this truy for basedir, then the method needs to be renamed and handle basedir only.

Reply:
Honestly, I don't understand the purpose of this at all:
https://github.com/apache/maven-verifier/blob/96c30e3cea783fcbb06963f98754a32148957f1b/src/main/java/org/apache/maven/shared/verifier/Verifier.java#L1418-L1422